uniswap-v2-core/test/VexchangeV2Factory.spec.ts

```diff
@@ -71,12 +80,14 @@ describe('VexchangeV2Factory', () => {
 
   it('retreivePairInitCode', async () => {
     // Retrieve the VexchangeV2Pair init-code from the factory
-    const initCode: BigNumber =  await factory.getPairInitHash()
+    const initCode: BigNumber = await factory.getPairInitHash()
 
     // Expected init-code (hard coded value is used in dependent modules as a gas optimisation, so also verified here).
     // Note: changing the hard-coded expected init-code value implies you will need to also update the dependency.
     // See dependency @ uniswap-v2-periphery/contracts/libraries/VexchangeV2Library.sol
-    expect(initCode, "VexchangeV2Pair init-code").to.eq('0x7e79aabe2533ac4a6df68a11c9e5389d4c11e8bec064a048ef999b7912cbb168')
+    expect(initCode, 'VexchangeV2Pair init-code').to.eq(
+      '0x7e79aabe2533ac4a6df68a11c9e5389d4c11e8bec064a048ef999b7912cbb168'
```

Comment:
Do the tests pass without updating this?

Reply:
Yup it does. This style change was made by the linter

Reply:
No the suggestion is fine, I just thought we changed V2Pair for some reason. We only changed V2Factory so ofc there was no change to the hash. All good here